packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, Wrapper, Menu } from 'react-aria-menubutton';
+import { FiDownload } from 'react-icons/fi';
+import { MdArrowDropDown } from 'react-icons/md';
+
+import styles from './Selector/Selector.module.css';
+
+interface Props<F extends string> {
+  formats: F[];
+  isSlice: boolean;
+  getFormatURL: (format: F) => string | undefined; // `undefined` if format is not supported
+}
+
+function ExportMenu<F extends string>(props: Props<F>) {
+  const { formats, isSlice, getFormatURL } = props;
+
+  if (!getFormatURL) {
```

Comment:
It would be nice to check also if `formats.map(getFormatURL)` gives defined values.

Otherwise, we can have an export menu with no entries.

Reply:
Yeah... That's what I had initially, but I found the code a little too complex compared to the likelihood of this happening. Perhaps we can revisit it if it ever happens?

Reply:
Well, now that we are knee-deep in this feature, we might as well implement the check and be done with it, no :smile: ?

Reply:
Haha fair, okay I'll put it back.

Reply:
Found a simpler way than what I had initially tried, so all good. Let me know what you think.